setup-kind/action.yaml

```diff
@@ -151,7 +151,6 @@ runs:
         # See: https://kubernetes.slack.com/archives/CEKK1KTN2/p1600268272383600
         kubeadmConfigPatches:
           - |
-            apiVersion: kubeadm.k8s.io/v1beta2
```

Comment:
Is this intentional?

Reply:
Yes - otherwise the patch doesn't work on clusters 1.23+

Reply:
ah, thanks for the clarification